keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -475,8 +483,12 @@ type Rule struct {
      // Required.
      Resources []string `json:"resources,omitempty" protobuf:"bytes,3,rep,name=resources"`
 
-     // Scope specifies the scope of this rule. If unspecified, the scope is
-     // not limited.
+     // Scope specifies the scope of this rule.
+     // Valid values are "Cluster", "Namespaced", and "*"
+     // "Cluster" means that only cluster-scoped resources will match this rule.
+     // "Namespaced" means that only namespaced resources will match this rule.
```

Comment:
Explicitly state whether namespaces are namespaced or cluster scoped.  Our authorizer treats them as namespaced as I recall.  I think most code treats them as cluster scoped.

Reply:
updated to explicitly state they are cluster scoped.